remote_file_system/client_interface.py

```diff
@@ -72,6 +71,8 @@ def _check_validity_on_server(self, file_path: Path) -> bool:
 
     def _get_file_from_server(self, file_path: Path) -> bytes:
         outgoing_message: Message = ReadFileRequest(request_id=uuid4(), filename=str(file_path))
+        # For testing only
+        # outgoing_message: Message = ReadFileRequest(request_id=UUID(int=0x12345678123456781234567812345678), filename=str(file_path))
```

Comment:
I think we don't have to commit the manual tests

Reply:
sure.